.github/workflows/tests.yaml

```diff
@@ -117,7 +123,13 @@ jobs:
     - name: Verify installing CRDs via kustomize works
       run: make install
 
+    - name: Load image to kind cluster
+      run: kind load docker-image gateway-operator:e2e-${{ github.sha }} --name chart-testing # It's the default name of the kind cluster via helm/kind-action
```

Comment:
Is it possible to use the output of `helm/kind-action` here? This should be clearer and do not need to change if the kind cluster name is changed in `helm/kind-action`.

Reply:
I'll directly specify the name used by helm/kind-action, so it won't rely on external sources anymore. WDYT?